rasa/core/slots.py

```diff
@@ -106,10 +107,10 @@ def __init__(
             )
 
         if initial_value is not None and not (min_value <= initial_value <= max_value):
-            logger.warning(
-                "Float slot ('{}') created with an initial value {}"
-                "outside of configured min ({}) and max ({}) values."
-                "".format(self.name, self.value, self.min_value, self.max_value)
+            warnings.warn(
```

Comment:
sometimes we have the space at the end of line / sometimes at the beginning of a line. It's not a big thing, but would be great if we are consistent.

Reply:
Agree (personally I think beginning of the line looks better), but I think it's also black messing it up

Reply:
Ok, looks like we mostly have it at the end so i'll try to conform to that